src/state/registers.ts

```diff
@@ -441,8 +441,9 @@ export abstract class RegisterSet implements vscode.Disposable {
    * The '"' (`dquote`) register, default register for edit operations and
    * mapped to the system clipboard by default.
    */
-  public dquote: Register = new GeneralPurposeRegister('"', "copy");
-
+  public get dquote(): Register {
+    return this._named.get("dquote") || new GeneralPurposeRegister('"', "copy");
```

Comment:
Since it's added in the constructor, I think it's fine to expect that it exists.
```suggestion
    return this._named.get("dquote")!;
```